docs/source/connections.rst

```diff
@@ -47,3 +47,14 @@ Precision             `"precision"`
 Enumeration Strings   `"enum_strs"`
 Engineering Units     `"units"`
 ===================== ===============
+
+
+Happi Plugin
+============
+
+The PyDM Plugin interface makes no mandate about the type of signals
+that we connect to our widgets. The :class:`.HappiPlugin` and corresponding
+``HappiChannel`` reimagines theses signals and sends entire ``ophyd`` objects
```

Comment:
Not sure what `reimagines` is getting at?

Reply:
Just that I'm not sending a value and instead an entire structure. Maybe tooting my own horn too much 😉